modules/management/kubectl-apply/variables.tf

```diff
@@ -41,15 +41,15 @@ variable "kueue" {
   description = "Install and configure [Kueue](https://kueue.sigs.k8s.io/docs/overview/) workload scheduler. A configuration yaml/template file can be provided with config_path to be applied right after kueue installation. If a template file provided, its variables can be set to config_template_vars."
   type = object({
     install              = optional(bool, false)
-    version              = optional(string, "v0.8.1")
+    version              = optional(string, "v0.9.0")
     config_path          = optional(string, null)
     config_template_vars = optional(map(any), null)
   })
   default = {}
 
   validation {
-    condition     = !var.kueue.install || contains(["v0.8.1"], var.kueue.version)
-    error_message = "Supported version of Kueue is v0.8.1"
+    condition     = !var.kueue.install || contains(["v0.9.0", "v0.8.1"], var.kueue.version)
+    error_message = "Default Supported version of Kueue is v0.9.0. Cluster toolkit also supports v0.8.1"
```

Comment:
nit: Update error message to just output supported versions.

Reply:
Updating this